.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
-sudo: false
-node_js:
-  - 0.10
-  - 0.12
+dist: trusty
+sudo: required
```

Comment:
For whatever reason the default Travis image takes forever to build the project with pulp. Even with the latest node (I tried 5.11 as well).